### ISSUE TITLE
ETIMEDOUT: connect ETIMEDOUT 50.19.103.36:5000

### ISSUE THREAD

**Issue by SINGHPRATAP** [sudharti/socify#25]
I am from INDIA. I am getting ETIMEDOUT: connect ETIMEDOUT 50.19.103.36:5000 while i trying heroku run bash command and same when trying heroku run rake db:migrate.
I also tried telnet rendezvous.runtime.heroku.com 5000 but it shows 
telnet: Unable to connect to remote host: Connection timed out.
Is it port block problem, if yes then how to solve it.

**Comment by scaffeinate** [comment]
@SINGHPRATAP Yes the port is being blocked by your Internet provider. Here is how to solve that http://stackoverflow.com/questions/15992689/operation-timed-out-connect2-errnoetimedout

**Comment by jkxruby** [comment]
i just meet the same question, and i make it ...just try :   heroku run:detached rake db:migrate    ,it  work well...
